transport/wsn/pom.xml

```diff
@@ -150,6 +150,12 @@
             <version>2.0.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.kaazing</groupId>
+            <artifactId>specification.x-kaazing-ping-pong</artifactId>
+            <version>develop-SNAPSHOT</version>
```

Comment:
Reminder (for you and me): fix version once k3po is released before merging this PR!